index.html

```diff
@@ -40,10 +46,20 @@ <h2 class="highligh-title">I'm a software developer</h2>
             /></a>
           </li>
           <li>
-            <a href="https://github.com/NatiDeme"
+            <a href="https://www.linkedin.com/in/natnael-demelash/"
+              ><img class="icons" src="images/Vector.png" alt="Linkedin"
+            /></a>
+          </li>
+          <li>
+            <a href="https://angel.co/u/natnael-demelash"
               ><img class="icons" src="images/github.png" alt="github"
             /></a>
           </li>
+          <li>
+            <a href="https://medium.com/@demelashnatnael"
+              ><img class="icons" src="images/Vectorm.png" alt="github"
+            /></a>
+          </li>
         </ul>
         <a href="#personal-works" id="arrow-icon"
           ><img src="images/Disabled.png" alt=""
```

Comment:
Kindly adjust your code and close "<a>" tag at its line. It shows your code cleaner. Also, please write some description of your image in "alt" property.